src/logger/infoLoggerMidelware.js

```diff
@@ -0,0 +1,19 @@
+import { methodInvocationLogger as logger }  from './logger'
+
+const LOG_MESSAGE_TEMPLATE = 'Method: %s, Status: %s, URL: %s, Body: %s, Params: %s, Query params: %s, Execution time: %sms'
+
+function logMethodInvocation (req, res, next) {
+  const startTime = Date.now()
```

Comment:
Let's use [process.hrtime.bigin](https://nodejs.org/docs/latest-v14.x/api/process.html#process_process_hrtime_time) to calculate execution time

Reply:
Fixed